README.md

```diff
@@ -1 +1,53 @@
 # Fisher's Manager README
+
+## How I operate
+
+### Feedback
+
+1. How do you like feedback - the medium (Slack, email, in-person, etc.)
+> Personally, I like feedback in-person. I'm also open to having tricky conversations over Slack, but when I said I want to take it offline, I ask that you respect that.
+2. How do you like feedback - routine like in 1:1s, or as-it-happens
+> Timely feedback is always better for me -- just grab me!
+3. How do you prefer to receive recognition? (public or private)
+> I'm totally fine with public recognition but don't expect it!
+
+### Goals and Support
+
+1. What makes 1:1s the most valuable for you?
+> 1:1s are a time for me to voice what I'm worried about.
```

Comment:
Does that mean the value for you comes from being heard? Or do you want to also collaborate with (or be coached by) your manager on these issues in your 1:1?

Reply:
Good question -- both? But sometimes I like to vent, which I think everyone does. I'll clarify.